jupyterhub_announcement/handlers.py

```diff
@@ -117,6 +115,11 @@ class AnnouncementUpdateHandler(AnnouncementHandler):
     async def post(self):
         """Update announcement"""
         user = self.get_current_user()
+        # Check if user is admin. If not raise a 403
+        if not user["admin"]:
+            raise web.HTTPError(
+                403, f"{user['admin']} is not authorized to update announcement"
```

Comment:
@mahendrapaipuri This will print `False is not authorized...` right?  Should it be `user["name"]`?

Reply:
Oh yeah, you are right. That is a typo. Cheers for catching it. Fixed it!